sys/riscv/riscv/trap.c

```diff
@@ -378,8 +383,15 @@ page_fault_handler(struct trapframe *frame, int usermode)
 	return;
 
 fatal:
+#if __has_feature(capabilities)
+	if (frame->tf_scause == SCAUSE_STORE_AMO_CAP_PAGE_FAULT) {
+		panictype = "capability ";
```

Comment:
1. It's prefix not the actual fault type (and it's not about the panic it's about the fault)
2. If you're going to use multiple full words, use _ to separate